cmd/manager/main.go

```diff
@@ -212,3 +224,23 @@ func setProfiler() {
 		}()
 	}
 }
+
+func lockHandler() (lockfile.Lockfile, error) {
+	lockFilePath, ok := os.LookupEnv("NMSTATE_INSTANCE_NODE_LOCK_FILE")
+	if !ok {
+		return "", errors.New("Failed to find NMSTATE_INSTANCE_NODE_LOCK_FILE ENV var")
+	}
+	log.Info(fmt.Sprintf("Try to take exclusive lock on file: %s", lockFilePath))
+	lockFileMutex, err := lockfile.New(lockFilePath)
+	if err != nil {
+		return lockFileMutex, errors.Wrapf(err, "failed to create lockFile for %s", lockFilePath)
+	}
+	for {
+		log.Info("try to get lock")
+		if err = lockFileMutex.TryLock(); err == nil {
+			log.Info(fmt.Sprintf("Successfully locked %s", lockFilePath))
+			return lockFileMutex, err
+		}
+		time.Sleep(10 * time.Second)
```

Comment:
Is not very robust to use an infinite loop here we need a timeout let's use `PollInmmediate` from https://godoc.org/k8s.io/apimachinery/pkg/util/wait.

I suppose we need a bit timeout since we are waitting for CNAO installed kubernetes-nmstate-handler to finish.